plugins/outputs/whatap/README.md

```diff
@@ -0,0 +1,21 @@
+# WhaTap Output Plugin
+
+This plugin writes to the WhaTap(<https://www.whatap.io>) APM(via TCP).
+
+```toml
+  ## You can create a project on the WhaTap site(https://www.whatap.io) 
+  ## to get license, project code and server IP information.
+
+  ## WhaTap license. Required
+  #license = "xxxx-xxxx-xxxx"
+
+  ## WhaTap project code. Required
+  #pcode = 1111

```

Comment:
Can we please use a more speaking name for the option 
```suggestion
  project_code = 1111

```

Reply:
Changed pcode to project_code.